src/controllers/BaseController.ts

```diff
@@ -59,7 +59,7 @@ export abstract class BaseController {
             };
         } else {
             options = {
-                headless: true,
+                headless: "new",
```

Comment:
This is to omit a warning https://developer.chrome.com/docs/chromium/new-headless